desertbot/modules/urlfollow/Mixer.py

```diff
@@ -0,0 +1,87 @@
+"""
+Created on May 5, 2019
+
+@author: MasterGunner
+"""
+from twisted.plugin import IPlugin
+from desertbot.moduleinterface import IModule
+from desertbot.modules.commandinterface import BotCommand
+from zope.interface import implementer
+
+from desertbot.message import IRCMessage
+from desertbot.utils.api_keys import load_key
+
+from twisted.words.protocols.irc import assembleFormattedText as colour, attributes as A
+
+import re
+
+
+@implementer(IPlugin, IModule)
+class Mixer(BotCommand):
+    def actions(self):
+        return super(Mixer, self).actions() + [('urlfollow', 2, self.follow)]
+
+    def help(self, query):
+        return 'Automatic module that follows Mixer URLs'
+
+    def onLoad(self):
+        self.mixerClientID = load_key('Mixer Client ID')
+
+    def follow(self, _: IRCMessage, url: str) -> [str, None]:
+        # Heavily based on Didero's DideRobot code for the same
+        # https://github.com/Didero/DideRobot/blob/06629fc3c8bddf8f729ce2d27742ff999dfdd1f6/commands/urlTitleFinder.py#L37
+        match = re.search(r'mixer\.com/(?P<mixerChannel>[^/]+)/?(\s|$)', url)
+        if not match:
+            return
+        channel = match.group('mixerChannel')
+
+        if self.mixerClientID is None:
+            return '[Mixer Client ID not found]'
+
+        chanData = {}
+        channelOnline = False
+        mixerHeaders = {'Accept': 'application/json',
+                         'Client-ID': self.mixerClientID}
```

Comment:
Indentation is a little off here. Looks good otherwise.

Reply:
Indentation is identical to the Twitch module, but I'm no pythonsm'n and could be missing something.

Reply:
You have a space too many it seems.